setup.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+setup(name='vSphere Automation SDK',
+      version='6.8.7',
```

Comment:
Can you use these values instead?

version='1.0.0', 
description='VMware vSphere Automation SDK for Python', 
author='VMware, Inc.'

Reply:
@tianhao64 Thanks for pointing this out! Suggested modifications are added in a rebased commit.